spec/gilded_rose_spec.js

```diff
@@ -1,7 +1,267 @@
+'use strict'
+
+function resetItems(){
+  items = []
+}
+
+function populateItems(){
+  items.push(new Item('+5 Dexterity Vest', 10, 20));
+  items.push(new Item('Aged Brie', 2, 0));
+  items.push(new Item('Elixir of the Mongoose', 5, 7));
+  items.push(new Item('Sulfuras, Hand of Ragnaros', 0, 80));
+  items.push(new Item('Backstage passes to a TAFKAL80ETC concert', 15, 20));
+  items.push(new Item('Conjured Mana Cake', 3, 20));
+}
+
 describe("Gilded Rose", function() {
+  describe("items have a sell-in value", () => {
+    it("should initialize a quality", function() {
+      resetItems()
+      populateItems()
+
+      expect(items[0].sell_in).toEqual(10)
+      expect(items[1].sell_in).toEqual(2)
+      })
+    })
+  describe("items have a quality value", () => {
+    it("should initialize a quality", () => {
```

Comment:
what functionality of the *implementation* is this testing? It seems the only reason this could break is if populateItems is changed.